src/azure-cli-core/azure/cli/core/tests/test_extension.py

```diff
@@ -230,6 +230,16 @@ def test_ext_compat_with_cli_require_ext_min_version(self):
             self.assertFalse(is_compatible)
             self.assertEqual(min_ext_required, expected_min_ext_required)
 
+    @mock.patch('sys.stdin.isatty', return_value=True)
+    def test_ext_dynamic_install_config_tty(self, _):
+        from azure.cli.core.extension.dynamic_install import _get_extension_use_dynamic_install_config
+        self.assertEqual(_get_extension_use_dynamic_install_config(None), 'yes_prompt')
+
+    @mock.patch('sys.stdin.isatty', return_value=False)
+    def test_ext_dynamic_install_config_tty(self, _):
```

Comment:
The name of the test should be `test_ext_dynamic_install_config_no_tty`? otherwise two tests are with the same name `test_ext_dynamic_install_config_tty`